repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -39,6 +45,20 @@
     ],
     "container": {
         "type": "DOCKER",
+{{#security.authn.enabled}}
+        "volumes": [
+            {
+                "containerPath": "{{security.authn.opt_mesosphere_dir}}",
+                "hostPath": "{{security.authn.opt_mesosphere_dir}}",
+                "mode": "RO"
+            },
+            {
+              "containerPath": "{{security.authn.dcos_service_account_credential_dir}}",
```

Comment:
Remove this containerPath (and also the relevant vars in config.json) once we can pass in service account creds as JSON blob using the secrets module.